unison-src/transcripts/fix3265.output.md

```diff
@@ -35,14 +35,12 @@ are three cases that need to be 'fixed up.'
             (w x ->
               let
                 use Nat + drop
-                f1 y =
-                  match y with
-                    0 -> w + x
-                    n -> 1 + f0 (drop y 1)
-                f0 y =
-                  match y with
-                    0 -> x
-                    n -> 1 + f1 (drop y 1)
+                f1 y = match y with
+                  0 -> w + x
+                  n -> 1 + f0 (drop y 1)
+                f0 y = match y with
+                  0 -> x
+                  n -> 1 + f1 (drop y 1)
```

Comment:
More use of soft hang, swinging the `match` up onto the starting line of the function.